internal/printer/pod.go

```diff
@@ -152,7 +168,7 @@ func createPodSummaryStatus(pod *corev1.Pod) (*component.Summary, error) {
 	sections.AddText("QoS", string(pod.Status.QOSClass))
 
 	if pod.DeletionTimestamp != nil {
-		summary.SetAlert(component.NewAlert(component.AlertStatusError, component.AlertTypeDefault, "Pod is being deleted", false, nil))
+		summary.SetAlert(component.NewAlert(component.AlertStatusError, component.AlertTypeDefault, "Pod is being deleted*****", false, nil))
```

Comment:
Do we need these additional asterisks?

Reply:
We definitely don't need it, was part of the testing, they are removed now